cinema/views.py

```diff
@@ -1 +1,56 @@
-# write views here
+from rest_framework import viewsets
+
+from cinema.models import Genre, Actor, CinemaHall, MovieSession, Movie
+from cinema.serializers import (
+    GenreSerializer,
+    ActorSerializer,
+    CinemaHallSerializer,
+    MovieSessionSerializer,
+    MovieSerializer,
+    MovieListSerializer,
+    MovieSessionListSerializer,
+    ActorListSerializer,
+    MovieCreateSerializer,
+    MovieSessionCreateSerializer,
+)
+
+
+class GenreViewSet(viewsets.ModelViewSet):
+    queryset = Genre.objects.all()
+    serializer_class = GenreSerializer
+
+
+class ActorViewSet(viewsets.ModelViewSet):
+    queryset = Actor.objects.all()
+
+    def get_serializer_class(self):
+        if self.action == "list":
+            return ActorListSerializer
+        return ActorSerializer
+
+
+class CinemaHallViewSet(viewsets.ModelViewSet):
+    queryset = CinemaHall.objects.all()
+    serializer_class = CinemaHallSerializer
+
+
+class MovieViewSet(viewsets.ModelViewSet):
+    queryset = Movie.objects.prefetch_related("actors", "genres")
+
+    def get_serializer_class(self):
+        if self.action == "list":
+            return MovieListSerializer
+        if self.action == "retrieve":
+            return MovieSerializer
+        return MovieCreateSerializer
```

Comment:
why do name this serializer as CreateSerializer, will it used only for creation ?